webpack/assets/javascripts/react_app/routes/Models/ModelsPage/index.js

```diff
@@ -26,6 +27,14 @@ const ModelsPage = () => {
       title: __('Hosts'),
     },
   };
+  const response = useAPI(
+    'get',
+    `${MODELS_API_PATH}?include_permissions=true`,
+    {
+      key: API_REQUEST_KEY,
+    }
+  );
+
```

Comment:
I didnt check the rest of the PR, but can we not move this out of the table? the reason for the table component is so other components will not handle api calls.
Also this change breaks webhooks which also use the table component (https://github.com/theforeman/foreman_webhooks/blob/master/webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/WebhooksIndexPage.js)

Reply:
Cant the code use redux selectors to access the `response`?

Reply:
That's a possibility, just seemed a bit less elegant to me

@parthaa thoughts?

Reply:
Fixed it in the latest commit. Secret sauce here is this line
```javascript
let response = useAPI(
    replacementResponse ? null : 'get',
    apiUrl.includes('include_permissions')
      ? apiUrl
      : `${apiUrl}?include_permissions=true`,
    {
      ...apiOptions,
      params: defaultParams,
    }
  );
```

Reply:
method being null implies that useAPI effects wont do anything.